### DISCUSSION TITLE
storage/hwameistor/resources/migrate/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1731] [Announcements]
# storage/hwameistor/resources/migrate/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/storage/hwameistor/resources/migrate/

**Comment by liyuerich** [comment]
还有数据卷转换， 能否也进行介绍？

**Comment by peterchenhc** [comment]
数据卷这一章可否添加创建的章节？

第一次创建pvc，出现pending状态，排查了许久，发现需要和pod关联之后调度了才会绑定pv
pvc的event："waiting for first consumer to be created before binding"；也不能有效在文档中搜索中到

**Reply by jzhupup** [reply]
https://docs.daocloud.io/kpanda/user-guide/storage/pvc/

在这里可以看到创建
